AstSemantics.md

```diff
@@ -198,12 +200,11 @@ is why it isn't the specified default.)
 
 ### Out of bounds
 
-The ideal semantics is for out-of-bounds accesses to trap. A module may
-optionally define that "out of bounds" includes low-memory accesses.
+The ideal semantics is for out-of-bounds accesses to trap, but the performance
+implications and ability to polyfill to JS are not yet fully clear.
```

Comment:
I agree. I think your rewording is fine if you take out "and ability to polyfill to JS".